dev-libs/libtracefs/files/libtracefs-1.3.1-musl-pthread.patch

```diff
@@ -0,0 +1,16 @@
+# On musl it is necessary to include pthread header else the compiler cannot
```

Comment:
Looks like this is upstream as https://git.kernel.org/pub/scm/libs/libtrace/libtracefs.git/commit/?id=3d1eb4b9bb2b8903345b8cf6af0168985b9002e1 and we could/should bump it.

Reply:
(But this PR is fine as it is too, just as a bonus)

Reply:
@thesamesam aha, thanks I'll close this one

Reply:
I can commit this one as it is, but maybe do another? 😁

Reply:
Yes I'll do a new one doing a version bump